src/XRP/model/xrp-issued-currency.ts

```diff
@@ -19,20 +21,32 @@ export default class XrpIssuedCurrency {
     issuedCurrency: IssuedCurrencyAmount,
   ): XrpIssuedCurrency | undefined {
     const currency = issuedCurrency.getCurrency()
-    const xrpCurrency = currency && XrpCurrency.from(currency)
+    if (!currency) {
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'Issued currency protobuf does not contain valid `currency` field.',
```

Comment:
```suggestion
        'IssuedCurrency protobuf does not contain `currency` field.',
```

Reply:
suggesting we always use the official name of the type when referring to the protobuf